src/pages/Mine/Mine.tsx

```diff
@@ -1,12 +1,113 @@
+import { Fieldset, Input } from "@react95/core";
+import React from "react";
+import { useCallback, useState } from "react";
+import { classNames } from "../../classNames";
+import Divider from "../../components/Divider/Divider";
 import { HackilyRewriteHistory } from "../../hooks";
 import styles from "./Mine.module.css";
 
+const INDIVIDUAL_MINING_RATE = 6;
+const CURRENT_DIFFICULTY = "5,731,203,885,580";
+const CURRENT_PROBABILITY = 0.0000000185185;
```

Comment:
Hey @adamsuskin! Just curious, but do you remember how you calculated this probability number?